.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows-latest, macos-latest, ubuntu-latest]
-        rust_version: [stable, 1.70.0]
+        rust_version: [stable, 1.71.0]
```

Comment:
What caused the need for the version bump?

Reply:
async-generic has a compile-time issue with 1.70.

Reply:
@gpeacock I was able to resolve the issue in `async-generic`, so I've now reverted the MSRV change.